src/Main.hs

```diff
@@ -83,6 +87,4 @@ isInteractive = do
 main = do
   -- Issue #201: disable backend when run in interactive mode
   isInt <- isInteractive
-  if isInt
-    then runAgda [Backend backend{isEnabled = const False}]
-    else runAgda [Backend backend]
+  runAgda [Backend backend | not isInt]
```

Comment:
I tried to test it, though I'm not 100% convinced that emacs did cooperate using `agda2hs` instead of agda. I'd appreciate other people trying it, though the change looks very reasonable (to me).

Reply:
Sad, thanks for checking! Will revert the change. I'm curious though, what's breaking when no backend is provided?

Reply:
You cannot do `C-c C-x C-x RET agda2hs` anymore (no backend names "agda2hs" was found), which is the whole point I guess.

Reply:
Oh, I wasn't even aware this was a command, everything makes sense now.

Reply:
Hmm, it seems this got merged after all, thus breaking interaction. I will fix shortly.